nautobot/ipam/tests/features/prefixes.feature

```diff
@@ -0,0 +1,134 @@
+      | 10.0.100.0/24 | CORP      | Network | Active | 10.0.0.0/8 |
+    And Nautobot shows the "Prefix Detail" view for the Prefix with prefix "10.0.100.0/24" and namespace "CORP"
+    And the view shows a "Utilization" of "0%"
+    And the table "Parent Prefixes" includes rows with the following properties:
+      | prefix     | type      |
+      | 10.0.0.0/8 | Container |
+
+    When I click the "Clone" button
+    Then Nautobot shows the "Add Prefix" view with the following presets:
+      | namespace | type    | status |
+      | CORP      | Network | Active |
+    When I make the following form entries:
+      | prefix      | type      |
+      | 10.0.0.0/16 | Container |
+    And I submit the form
+    Then a Prefix exists in Nautobot with the following properties:
+      | prefix      | namespace | type      | status | parent     |
+      | 10.0.0.0/16 | CORP      | Container | Active | 10.0.0.0/8 |
+    And Nautobot shows the "Prefix Detail" view for the Prefix with prefix "10.0.0.0/16" and namespace "CORP"
+    And the table "Parent Prefixes" includes rows with the following properties:
+      | prefix     | type      |
+      | 10.0.0.0/8 | Container |
+    And the tab "Child Prefixes" includes a badge with the number "1"
+
+    When I click the "Child Prefixes" tab
+    Then Nautobot shows the "Child Prefixes" tab
+    And the table "Child Prefixes" includes rows with the following properties:
+      | prefix        | type    | status |
+      | 10.0.0.0/18   | —       | —      |
+      | 10.0.64.0/19  | —       | —      |
+      | 10.0.96.0/22  | —       | —      |
+      | 10.0.100.0/24 | Network | Active |
+
+    When I click the "10.0.96.0/22" link
+    Then Nautobot shows the "Add Prefix" view with the following presets:
+      | prefix        | namespace |
+      | 10.0.96.0/22  | CORP      |
+    When I make the following form entries:
+      | type         | type      | status |
+      | 10.0.96.0/20 | Container | Active |
+    And I submit the form
+    Then a Prefix exists in Nautobot with the following properties:
+      | prefix       | namespace | type      | status | parent      |
+      | 10.0.96.0/20 | CORP      | Container | Active | 10.0.0.0/16 |
+    And Nautobot shows the "Prefix Detail" view for the Prefix with prefix "10.0.96.0/20" and namespace "CORP"
+    And the view shows a "Utilization" of "6%"
+    And the table "Parent Prefixes" includes rows with the following properties:
+      | prefix      | type      | status |
+      | 10.0.0.0/8  | Container | Active |
+      | 10.0.0.0/16 | Container | Active |
+    And the tab "Child Prefixes" includes a badge with the number "1"
+
+    When I navigate to the "Prefix List" view
+    And specify a filter of "?namespace=CORP"
+    Then the table "Prefixes" includes rows with the following properties:
+      | prefix        | namespace | type      | status | children | utilization |
+      | 10.0.0.0/8    | CORP      | Container | Active | 3        | 0%          |
```

Comment:
I am not sure how utilization is calculated here. But with 3 children utilization should be > 0%? Ignore this if I am completely wrong.

Reply:
A /16 allocated out of a /8 is 1/256 utilization which rounds down to 0%. Probably would be good to add a comment here though.